docs/Reference/Errors.md

```diff
@@ -230,6 +230,11 @@ A callback for a hook timed out
 
 The logger accepts either a `'stream'` or a `'file'` as the destination.
 
+#### FST_ERR_LOG_INVALID_OBJECT
+<a id="FST_ERR_LOG_INVALID_OBJECT"></a>
+
+The logger should have a `'child'` method that allows the creation of stateful loggers.
```

Comment:
I would list all the method that are missing

Reply:
Yeah. What if i just pass console as logger. Sure it misses child but also fatal